Frameworks/UIKit/WYPopoverController.mm

```diff
@@ -2634,8 +2639,12 @@ - (void)dismissPopoverAnimated:(BOOL)aAnimated
         }
 #endif
     };
+
+    UIPopoverPresentationController* popoverPresentationController = [viewController popoverPresentationController];
     
     void (^completionBlock)() = ^() {
+        // Keep popoverPresentationController alive via block-capture during teardown.
+        (void)popoverPresentationController;
```

Comment:
although this is probably fine - i just kicked off a CI build

---
In reply to: [109265389](https://github.com/Microsoft/WinObjC/pull/2195#discussion_r109265389) [](ancestors = 109265389,109064009)